tests/quantization/autoawq/test_awq.py

```diff
@@ -104,15 +107,41 @@ def test_quantized_model_conversion(self):
         """
         from awq.modules.linear import WQLinear_GEMM, WQLinear_GEMV
 
-        self.tokenizer(self.input_text, return_tensors="pt").to(torch_device)
+        from transformers.integrations.awq import replace_with_awq_linear
+
+        model_id = "facebook/opt-350m"
+        config = AutoConfig.from_pretrained(model_id, revision="cb32f77e905cccbca1d970436fb0f5e6b58ee3c5")
+        quantization_config = AwqConfig(bits=4)
+
+        with init_empty_weights():
```

Comment:
This test will need a `@require_accelerate` decorator if this is used

Reply:
The `require_accelerate` decorator is already being set at `AwqTest` class